test/trampoline.js

```diff
@@ -49,4 +52,29 @@ describe('trampoline', function () {
       }
     );
   });
+
+  it('should support currying', function () {
+    const isArray = RA.trampoline(Array.isArray);
+
+    assert.isTrue(isArray([]));
+    assert.isFalse(isArray(() => {}));
+
+    const max = RA.trampoline(Math.max);
+
+    assert.equal(max(1, 2, 3), 3);
+  });
+
+  it('should support placeholder to specify "gaps"', function () {
```

Comment:
Please drop this test. We only use placeholder tests for function with arity = 1. For other function we exclusively use `should support currying tests`.